### PR TITLE
Issue18

### LINKED ISSUE #18: 🔥 docs error
I'm trying to reproduce the example figure (which comes from https://commensurability.readthedocs.io/en/latest/tessellation/usage/analysis/rotating_bar/#adding-a-rotating-bar-to-a-potential) so I'm running that from the top.

I'm encountering the error:

```python
tanal = TessellationAnalysis2D(
    initial_condition,
    values,
    potential_definition,
    dt,
    steps,
    pattern_speed=omega,
    chunksize=500,
    mpchunksize=20,
)
```

produces the error `TypeError: MPAnalysisBase.__init__() got an unexpected keyword argument 'chunksize'`

Please update the docs to fix this example.

— openjournals/joss-reviews#7009

### PR DESCRIPTION
**Pull request by michael-petersen**
I've tested the proposed fix for `chunksize` in the docs and this works on my machine.

cf. https://github.com/ilikecubesnstuff/commensurability/issues/18#issuecomment-2302642357

I've also checked for the same issue in `examples`; it isn't there but I did have an issue with multiprocessing that specifying the namespace fixes (on my machine).

### CODE REVIEW

**Comment by michael-petersen** [comment]
Just to also note, I've branched off of `joss`, but I think this could be cherry-picked into `main` without much trouble.

File: docs/tessellation/usage/analysis/rotating_bar.md [hunk 1]
```diff
@@ -57,8 +57,16 @@ dt = 0.01 * u.Gyr
...
+    initial_condition,
+    values,
+    potential_definition,
+    dt,
+    steps,
+    pattern_speed=omega,
+    pidgey_chunksize=500,
+    mp_chunksize=20,
+)
 tanal.save("no_bar_example.hdf5")
```
Comment (michael-petersen) [inline docs/tessellation/usage/analysis/rotating_bar.md:70]: I did have some trouble with this line in a couple tests (in retrieving the source code to save into the output) -- but thinking it was specific to my build. Anyone else find an issue? If so, I'll report more formally.

**Comment by ilikecubesnstuff** [comment]
Thank you for the pull request! The changes look good to me.

File: docs/tessellation/usage/analysis/rotating_bar.md (hunk 1 above)
Comment (ilikecubesnstuff) [reply docs/tessellation/usage/analysis/rotating_bar.md:70]: What tests were causing issues?

**Pull request merged**